### PR TITLE
Boto3 support

### PR DESCRIPTION
**Pull request by deargle**

### CODE REVIEW

File: psiturk/amt_services.py [hunk 1]
```diff
@@ -56,26 +51,24 @@
...
+PERCENT_ASSIGNMENTS_APPROVED_QUAL_ID = '000000000000000000L0'
+NUMBER_HITS_APPROVED_QUAL_ID = '00000000000000000040'
+LOCALE_QUAL_ID = '00000000000000000071'
+MASTERS_QUAL_ID = '2F1QJWKUDD8XADTFD2Q0G6UTO95ALH'
+MASTERS_SANDBOX_QUAL_ID = '2ARFPLSP75KLA8M8DH1HTEQVJT3SY6'
+
+NOTIFICATION_VERSION = '2006-05-05'
 
-class MTurkHIT(object):
+qqclass MTurkHIT(object):
```
Comment (deargle) [inline psiturk/amt_services.py]: @shimeez what is a `qqclass`?

Comment (shimeez) [reply psiturk/amt_services.py]: Apologies, must have accidentally tried to close vim and saved it. I've updated the branch

**Comment by deargle** [comment]
vim is life

**Comment by deargle** [comment]
Ayy easy there with those force pushes

**Comment by deargle** [comment]
closed by 986523ee37bc07212b4a2906baf39326e4aeb412

Thanks @shimeez ! I'm not the best with git. I worked on integrating your fork offline, and then I merged it in

**Pull request closed without merging**